code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -1195,13 +1195,83 @@
 /*
 /obj/item/ammo_magazine/sbaw/he
 	name = "20mm SBAW magazine (HEFI)"
-	icon_state = "sbaw_he"
+	icon_state = "sbaw_incend"
 	ammo_type = /obj/item/ammo_casing/shotgun/payload
 */
 // SCI ammo
+/obj/item/ammo_magazine/pistol_35/laser
+	name = "standard magazine (9mm LASER)"
+	desc = "A 10 round magazine marked for 9mm standard. This one is intended for SI LASER ammunition."
+	icon_state = "pistol_lsr"
+	ammo_type = /obj/item/ammo_casing/pistol_35/laser
+
+/obj/item/ammo_magazine/pistol_35/laser/empty
+	desc = "A 10 round magazine marked for 9mm standard. This one is not pre-filled, and is intended for SI LASER ammunition."
+	initial_ammo = 0
+
+/obj/item/ammo_magazine/smg_35/laser
+	name = "smg magazine (9mm LASER)"
+	desc = "A 32 round magazine marked for 9mm smg. This one is intended for SI LASER ammunition."
+	icon_state = "smg_lsr"
+	ammo_color = ""
+	ammo_type = /obj/item/ammo_casing/pistol_35/laser
+
+/obj/item/ammo_magazine/smg_35/laser/empty
+	desc = "A 32 round magazine marked for 9mm smg. This one is not pre-filled, and is intended for SI LASER ammunition."
+	initial_ammo = 0
+
+/obj/item/ammo_magazine/magnum_40/laser
+	name = "magazine (10mm Magnum LASER)"
+	desc = "A 10 round magazine marked for 10mm Magnum. This one is intended for SI LASER ammunition."
+	icon_state = "magnum_lsr"
+	ammo_type = /obj/item/ammo_casing/magnum_40/laser
+
+/obj/item/ammo_magazine/magnum_40/laser/empty
+	desc = "A 10 round magazine marked for 10mm Magnum. This one is not pre-filled, and is intended for SI LASER ammunition."
+	initial_ammo = 0
+
+/obj/item/ammo_magazine/kurtz_50/laser
+	name = "heavy pistol magazine (12mm LASER)"
+	desc = "A 7 round magazine marked for 12mm. This one is intended for SI LASER ammunition."
+	icon_state = "kurtz_lsr"
+	ammo_type =
```

Comment:
this is missing ammo type

Reply:
yeah, i was fixing it as you wrote it. it SHOULD be fixed now.

Reply:
_/obj/item/ammo_magazine/ammobox/kurtz_50/laser_
is the correct path

Reply:
actually no
it's /obj/item/ammo_casing/kurtz_50/laser and i was totally wrong
fixing now